client/manipulators/placement.hpp

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+#include <algorithm>
+
+#include "engine-entities.hpp"
+#include "graphics/renderable.hpp"
+#include "resources/mesh.hpp"
+
+namespace tec {
+using RenderableMap = Multiton<eid, Renderable*>;
+namespace manipulator {
+
+const float MAX_DISTANCE = 25.0f; // Maximum distance at which to place.
```

Comment:
it would be fantastic if this came from the global config file

Reply:
Yep there is already a config I can move this to.

Reply:
Added to config file `max_placement_distance`